corehq/toggles/__init__.py

```diff
@@ -1115,6 +1115,7 @@ def _ensure_valid_randomness(randomness):
     parent_toggles=[SYNC_SEARCH_CASE_CLAIM]
 )
 
+# TODO: GA or Bust: See CaseSearchConfig.dynamically_update_results
```

Comment:
will split screen feature work if this linked feature flag is not enabled?
I wonder if both need to be GA'ed together.

Reply:
The Split Screen UI will work if this flag is not enabled -- it just adds functionality to the Split Screen UI.